app/src/main/java/com/example/dungeontest/model/EditorViewModel.kt

```diff
@@ -5,39 +5,7 @@ import androidx.lifecycle.AndroidViewModel
 
 class EditorViewModel( application: Application): AndroidViewModel(application) {
```

Comment:
can we please delete this, now that it's no longer in use?

Reply:
Addressed albeit a touch reluctantly. It just seems more intuitive for me that every screen has a corresponding viewModel. If data needs to be shared between screens, then we can have a third view model to facilitate the process. I'm  leaning towards this design philosophy in an attempt to honor the whole separation of concerns framework.

Reply:
I'm not sure I share that intuition. What about the fact that online sources about sharing complex data (not simply a string or number) between jetpack-compose screens recommend using a ViewModel?
And the fact that SettingsViewModel is being used by both the SettingsScreen and the MainScreen?

Reply:
In any case, I'll close this thread as soon as everything else is closed and we're ready to merge